resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,411 @@
 @extends('app')
-@section('title', 'Generate OFP')
+@section('title', 'SimBrief Flight Planning')
 
 @section('content')
+
+  @php
+    $loadmin = $flight->load_factor - $flight->load_factor_variance ;
+    $loadmax = $flight->load_factor + $flight->load_factor_variance ;
+    if($loadmin < 1) { $loadmin = 1 ; }
+    if($loadmax > 100) { $loadmax = 100 ; }
+  @endphp
+
+  @foreach($aircraft as $acdetails)
+    @php
+      $simbrieftype = $acdetails->icao ;
+      $subflid = $acdetails->subfleet_id ;
+      if($acdetails->icao == 'A20N') { $simbrieftype = 'A320' ; }
+      if($acdetails->icao == 'A21N') { $simbrieftype = 'A321' ; }
+      if($acdetails->icao == 'B77L') { $simbrieftype = 'B77F' ; }
+      if($acdetails->icao == 'B773') { $simbrieftype = 'B77W' ; }
+      if($acdetails->icao == 'E35L') { $simbrieftype = 'E135' ; }
```

Comment:
What about additional types? This will only set the `$simbrieftype` based on the last aircraft in the list

Reply:
$simbrieftype = $acdetails->icao ;

It is already being assigned before the if's (just 2 lines above the first if) , these are the changes where we make the icao type compatible with simbrief ... They are the ones I found where simbrief uses different type codes for an icao code